.github/workflows/build_portable_linux_pytorch_wheels.yml

```
@@ -203,7 +224,79 @@ jobs:
     with:
       amdgpu_family: ${{ inputs.amdgpu_family }}
       test_runs_on: ${{ needs.generate_target_to_run.outputs.test_runs_on }}
-      cloudfront_url: ${{ inputs.cloudfront_url }}
+      cloudfront_url: ${{ inputs.cloudfront_staging_url }}
       python_version: ${{ inputs.python_version }}
       torch_version: ${{ needs.build_pytorch_wheels.outputs.torch_version }}
       pytorch_version: ${{ inputs.pytorch_version }}
+
+  upload_pytorch_wheels:
+    name: Upload PyTorch Wheels to S3
```

Nit: I would revise the job name as wheels have already been uploaded.
```suggestion
    name: Release PyTorch Wheels to S3
```